components/templates/src/filters.rs

```diff
@@ -72,13 +75,32 @@ pub fn base64_decode<S: BuildHasher>(
     Ok(to_value(&String::from_utf8(decode(s.as_bytes()).unwrap()).unwrap()).unwrap())
 }
 
+pub fn num_format(value: &Value, args: &HashMap<String, Value>) -> TeraResult<Value> {
+    use num_format::{Locale, ToFormattedString};
+
+    let num = try_get_value!("num_format", "value", i64, value);
+
+    let locale = match args.get("locale") {
+        Some(locale) => try_get_value!("num_format", "locale", String, locale),
+        None => "en".to_string(),
+    };
```

Comment:
I think making this field required would be nicer, or pass the config default_language to the filter. No reasons to default to `en` if we don't know anything about the current locale.

Reply:
I opted to make it default to the `default_language` in the config, as I think it feels more ergonomic than having to pass a locale every time.